### PR TITLE
Prepare for 2.0.10 release

### PR DESCRIPTION
**Pull request by scardena**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # 2.0.9 (2023-06-06)
 
+* Add startupProbe (#97)
+
+# 2.0.9 (2023-06-06)
```
Comment (ctffarley) [inline CHANGELOG.md:5]: The `# 2.0.9` heading is in here twice, I think this should be 2.0.10 once

**Review by ctffarley — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (scardena) [reply CHANGELOG.md:5]: yup, sorry. fixed

Comment (ctffarley) [reply CHANGELOG.md:5]: I should've put this in the comment also but the date is copy pasted and should be 2023-09-25

**Review by ctffarley — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (scardena) [reply CHANGELOG.md:5]: I noticed that too, I fixed it right after.

**Pull request merged**